roxie/ccd/ccdcontext.cpp

```diff
@@ -2738,7 +2738,13 @@ class CRoxieServerContext : public CRoxieContextBase, implements IRoxieServerCon
             {
                 // MORE - can we accumulate results from several runs?
                 statsWu.setown(daliHelper->createWorkUnit());
-                queryExtendedWU(statsWu)->copyWorkUnit(_factory->queryWorkUnit(), false, true);
+                //queryExtendedWU(statsWu)->copyWorkUnit(_factory->queryWorkUnit(), false, true);
+                StringBuffer dllFileName;
+                splitFilename(_factory->queryDll()->queryName(), nullptr, nullptr, &dllFileName, nullptr, false);
+#ifndef _WIN32
```

Comment:
trivial: I was going to suggest making this unconditional. However replaceString should also check that lenOldStr != 0.